src/services/user.service.ts

```diff
@@ -59,14 +59,8 @@ class UserService implements UserServiceInterface {
         login: User['login'],
         password: User['password']
     ): Promise<UserAccessToken> {
-        const user: Model<User> | null = await this.userModel.findOne(
-            findUser({ login })
-        );
-
-        const isValidPass = await comparePassword(
-            password,
-            user?.getDataValue('password')
-        );
+        const user: any = await this.userModel.findOne(findUser({ login }));
+        const isValidPass = await comparePassword(password, user?.password);
```

Comment:
Hmm. Just remembered doing this right after making the PR. 
 
I set this to 'any' because the tests would fail as 'getDataValue' returned 'not a function' =/. I'm going to take a look now and see if there's a workaround.

Reply:
Thankfully that wasn't a difficult fix :)